PHP / Laravel updates (#4422)
getsentry/sentry-docs: src/platforms/php/guides/laravel/index.mdx

description: "Learn about using Sentry with Laravel."
---

> Laravel is supported via a native package: [sentry-laravel](https://github.com/getsentry/sentry-laravel).

Reviewer: ```suggestion
Laravel is supported using a native package: [sentry-laravel](https://github.com/getsentry/sentry-laravel).
```

}
```

> ### Verify with Artisan

Reviewer: ```suggestion
### Verify With Artisan
```

You can test your configuration using the provided `sentry:test` artisan command:

```shell
php artisan sentry:test
```

> ### Verify with code

Reviewer: ```suggestion
### Verify With Code
```

Visiting this route will trigger an exception that will be captured by Sentry.

> ## Local development and testing

Reviewer: ```suggestion
## Local Development and Testing
```

If you do leave Sentry enabled when developing or running tests it's possible it has a negative effect on the performance of your application or test suite.

## Performance Monitoring

Set `traces_sample_rate` in `config/sentry.php` or `SENTRY_TRACES_SAMPLE_RATE` in your `.env` to a value greater than `0.0`. Setting a value greater than `0.0` will enable Performance Monitoring, `0` (the default) will disable Performance Monitoring.

```shell {filename:.env}
# Be sure to lower this value in production otherwise you could burn through your quota quickly.
SENTRY_TRACES_SAMPLE_RATE=1.0
```

> The example configuration above will transmit 100% of captured traces. Be sure to lower this value in production otherwise you could burn through your quota quickly.

Reviewer: ```suggestion
The example configuration above will transmit 100% of captured traces. Be sure to lower this value in production or you could use up your quota quickly.
```

> You can also be more granular with the sample rate by using the `traces_sampler` option, learn more about sampling in [Using Sampling to Filter Transaction Events](configuration/filtering/#using-sampling-to-filter-transaction-events).

Reviewer: ```suggestion
You can also be more granular with the sample rate by using the `traces_sampler` option. Learn more in [Using Sampling to Filter Transaction Events](configuration/filtering/#using-sampling-to-filter-transaction-events).
```
